backend/pom.xml

```diff
@@ -29,6 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+		</dependency>
```

Comment:
By adding liquibase in this way you are using it as a part of your code. It is not a best practice to use liquibase programmatically or make your actual service aware of utility tools such as liquibase.

Please don't use liquibase as source dependncy. Mavan should use liquibase as a plugin.

Example can be found here: https://rcarrillo.dev/setup-liquibase-on-maven/

Reply:
Done